src/app/core/teams/teams.service.js

```diff
@@ -753,6 +805,8 @@ const updateTeams = async (user) => {
 	user.teams = updatedTeams;
 };
 
+const isObjectIdEqual = (value1, value2) => value1.equals(value2);
```

Comment:
Is it possible for value1 to come in as `null` or `undefined` here?

Reply:
Updated with null check just to be safe.